local_units/permissions.py

```diff
@@ -38,3 +40,56 @@ def has_object_permission(self, request, view, obj):
             or get_local_unit_region_validators(obj).filter(id=user.id).exists()
             or get_local_unit_global_validators(obj).filter(id=user.id).exists()
         )
+
+
+class ExternallyManagedLocalUnitPermission(permissions.BasePermission):
+    def has_permission(self, request, view):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+
+        return request.user.is_superuser
+
+
+class BulkUploadValidatorPermission(permissions.BasePermission):
+    message = "You do not have permission to create bulk uploads for this country and local unit type."
+
+    def has_permission(self, request, view):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+
+        user = request.user
+        if user.is_superuser:
+            return True
+
+        if not isinstance(request.data, dict):
+            return True
+
+        country_id = request.data.get("country")
+        local_unit_type_id = request.data.get("local_unit_type")
+
+        if not country_id or not local_unit_type_id:
+            return True
```

Comment:
country_id and local_unit_type_id are required otherwise denied.

```suggestion
        if not country_id and not local_unit_type_id:
            return False
```

Also we need this in serializer.